source/features/view-last-pr-deployment.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+import {LinkExternalIcon} from '@primer/octicons-react';
+
+import features from '.';
+import onConversationHeaderUpdate from '../github-events/on-conversation-header-update';
+
+const deploymentSelector = '.js-timeline-item [data-url$="deployed"] .TimelineItem-body .btn[target="_blank"]';
+
+function init(): void | false {
+	if (select.exists('.rgh-last-deployment')) {
+		return false;
+	}
```

Comment:
-> `exclude: […]`

Reply:
The exists is do to init being called 2 times.

Reply:
That's what `exclude` is for. I think:

```js
	exclude: [
 		() => !select.exists(deploymentSelector),
		() => select.exists('.rgh-last-deployment')
 	],
```

Or a single function, if you figure out the correct boolean logic 😃

Reply:
From what I understand exclude only runs once here since we are using `observeElement`, and its not the feature loader is not calling the init.

Please correct me if I am wrong

Reply:
😱 

I was totally sure `additionalListeners` still checked `include`/`exclude`, but they don't 👍

Reply:
Note to self remove the false. (We only add the false for the logger, here it would have ran already right?)